plugins/CustomDimensions/tests/System/expected/test___API.getReportMetadata_day.xml

```diff
@@ -29,7 +29,7 @@
 		<metricTypes>
 			<nb_visits>number</nb_visits>
 			<nb_actions>number</nb_actions>
-			<avg_time_on_site>duration</avg_time_on_site>
+			<avg_time_on_site>duration_s</avg_time_on_site>
```

Comment:
the change in this type name is ok for Looker Studio? Same for money <-> currency.

Reply:
The name values are irrelevant, they can easily be converted to what is expected within Looker Studio.